docs/api/console.md

```diff
@@ -0,0 +1,123 @@
+# 🎮 console
+
+An implementation of the standard [`console`](https://developer.mozilla.org/en-US/docs/Web/API/console) API.
+
+## Quick start
+
+The console API provides access to the `jztz` debug log. 
+General purpose mesages can be created using [`console.log()`](#log)
```

Comment:
mesages -> messages
created -> logged (console doesn't create messages, it logs them)